.buildkite/bk.integration.pipeline.yml

```diff
@@ -78,6 +84,9 @@ steps:
           provider: "gcp"
           machineType: "n1-standard-8"
```

Comment:
nit: in a future PR let's get rid of those ancient n1 instance types in favor of n2.